### ISSUE TITLE
filename_prefix not applied to netCDF output

### ISSUE THREAD

**Issue by bossie** [Open-EO/openeo-geopyspark-driver#876]
This process graph specifies a `filename_prefix` but will output an `openEO.nc` file instead.

```json
{
  "process_graph": {
    "applydimension1": {
      "arguments": {
        "data": {
          "from_node": "loadcollection1"
        },
        "dimension": "bands",
        "process": {
          "process_graph": {
            "arrayappend1": {
              "arguments": {
                "data": {
                  "from_parameter": "data"
                },
                "value": {
                  "from_node": "divide1"
                }
              },
              "process_id": "array_append",
              "result": true
            },
            "arrayelement1": {
              "arguments": {
                "data": {
                  "from_parameter": "data"
                },
                "index": 1
              },
              "process_id": "array_element"
            },
            "arrayelement2": {
              "arguments": {
                "data": {
                  "from_parameter": "data"
                },
                "index": 0
              },
              "process_id": "array_element"
            },
            "divide1": {
              "arguments": {
                "x": {
                  "from_node": "arrayelement1"
                },
                "y": {
                  "from_node": "arrayelement2"
                }
              },
              "process_id": "divide"
            }
          }
        }
      },
      "process_id": "apply_dimension"
    },
    "loadcollection1": {
      "arguments": {
        "bands": [
          "VV",
          "VH"
        ],
        "id": "SENTINEL1_GLOBAL_MOSAICS",
        "spatial_extent": {
          "crs": "EPSG:32755",
          "east": 576780,
          "north": 5246260,
          "south": 5244980,
          "west": 575500
        },
        "temporal_extent": [
          "2019-12-31T23:59:59Z",
          "2021-01-01T00:00:00Z"
        ]
      },
      "process_id": "load_collection"
    },
    "renamelabels1": {
      "arguments": {
        "data": {
          "from_node": "applydimension1"
        },
        "dimension": "bands",
        "target": [
          "VV",
          "VH",
          "VHdivVV"
        ]
      },
      "process_id": "rename_labels"
    },
    "saveresult1": {
      "arguments": {
        "data": {
          "from_node": "renamelabels1"
        },
        "format": "NetCDF",
        "options": {
          "filename_prefix": "lcfm-s1-55GEN_062_19"
        }
      },
      "process_id": "save_result",
      "result": true
    }
  }
}
```

**Comment by bossie** [comment]
Probably already fixed in 25a420b22c8df294023ad297364b831ca6ee1a97: TBC.

**Comment by bossie** [comment]
Confirmed: out came `lcfm-s1-55GEN_062_19.nc`.
